README.md

```diff
@@ -1,2 +1,38 @@
 # lake-temperature-lstm-static
-Predict lake temperatures at depth using static lake attributes
+
+Predict lake temperatures at multiple depths using an LSTM trained to make use of static lake attributes.
+
+## Installing
+
+Create a new `conda` environment by running
+
+```
+conda update -n base conda
+conda env create -f environment.yaml
+conda activate ltls
+```
+
+Note: if [Mamba](https://github.com/mamba-org/mamba) is available, the environment may be created significantly faster by running
+
+```
+mamba update -n base mamba
```

Comment:
I didn't know about `mamba`. That looks helpful!

Reply:
I just started using `mamba`. It was the `Snakemake` docs that pointed me to it. It is quick, especially when it comes to resolving package conflicts, and it's been easy to use as a drop-in replacement for `conda` in most circumstances.